install/run_install.sh

```diff
@@ -326,29 +313,43 @@ if [ "$SELECT_TOR" = "--select-tor" ] ; then
 	    sudo rm -r ~/debian-packages
 	  else number_torversion=0 ; fi
 	fi
-	if [ $number_torversion = 0 ] ; then
+	if [ $number_torversion = 0 ]; then
 	  echo -e "${WHITE}[!]         Something didn't go as expected!${NOCOLOR}"
 	  echo -e "${WHITE}[!]         I will try to install the latest stable version.${NOCOLOR}"
 	fi
 else number_torversion=0 ; fi
 
-# 6b. Compile and install the latest stable Tor version
-if [ $number_torversion = 0 ] ; then
-	mkdir ~/debian-packages; cd ~/debian-packages
-	apt source tor
-	sudo apt-get -y install fakeroot devscripts
-	#sudo apt-get -y install tor deb.torproject.org-keyring
-	#sudo apt-get -y upgrade tor deb.torproject.org-keyring
-	sudo apt-get -y build-dep tor deb.torproject.org-keyring
-	cd tor-*
-	sudo debuild -rfakeroot -uc -us
-	cd ..
-	sudo dpkg -i tor_*.deb
+# 5b. Compile and install the latest stable Tor version
+if [ $number_torversion = 0 ]; then
+  LATEST_STABLE_VERSION=$(curl --silent $TORURL | grep "/torproject/tor/releases/tag/" | sed -e "s/<a href=\"\/torproject\/tor\/releases\/tag\/tor-//g" | sed -e "s/\">//g" | grep -v "-" 2>&1 | head -n 1)
```

Comment:
> This will not always get the right version. For example, if they release a new 0.4.7.1-alpha-dev, it will be on the top of the list. Possible solution: exclude all -alpha and -rc, sort the list and head -n 1

Sorted by version for only stable versions:
```
array=$(curl --silent $TORURL | grep "/torproject/tor/releases/tag/" | sed -e "s/<a href=\"\/torproject\/tor\/releases\/tag\/tor-//g" | sed -e "s/\">//g" | grep -v "-"); IFS=$'\n' sorted=($(sort -r <<<"${array[*]}")); unset IFS; printf "[%s]\n" "${sorted[@]}" | sed -e 's/.$//' | cut -c8-
```
```
0.4.6.6
0.4.6.5
0.4.5.9
0.4.5.8
0.4.4.9
0.3.5.15
```